.github/workflows/pytest-workflow.yml

```diff
@@ -122,7 +122,8 @@ jobs:
             tags: subworkflows/bcl_demultiplex
           - profile: "conda"
             tags: subworkflows/vcf_annotate_ensemblvep
-
+          - profile: "conda"
+            tags: subworkflows/fasta_clean_fcs
```

Comment:
Why is this here?

Reply:
I don't know, this is created automatically. Good catch, I removed it.